src/ui/button/props.js

```diff
@@ -106,32 +105,38 @@ export type ButtonProps = {|
     scopes : $ReadOnlyArray<string>,
     responseType : string,
     billingOptions : BillingOptions,
-    state? : string
-|};
-
+    state? : string,
+    cspNonce? : {|
+        nonce? : string
+    |},
+    returnurl? : string
+   |};
+   
 export type ButtonPropsInputs = {|
     clientID : string,
     fundingSource? : $Values<typeof FUNDING>,
     style? : ButtonStyleInputs | void,
     locale? : $PropertyType<ButtonProps, 'locale'> | void,
     env? : $PropertyType<ButtonProps, 'env'> | void,
-    meta? : $PropertyType<ButtonProps, 'meta'> | void,
+    // meta? : $PropertyType<ButtonProps, 'meta'> | void,
```

Comment:
Can we delete `meta` if it's not needed?
```suggestion
```

Reply:
done